src/ci/bin/testSingleWorkflowRunner.sh

```diff
@@ -68,3 +68,23 @@ jq '{
 
 cmp <(jq -cS . actual.json) <(jq -cS . expected.json)
 popd > /dev/null
+
+# Test 3: program should exit with error in case if validation of command line arguments failed
+java -jar "${CROMWELL_BUILD_CROMWELL_JAR}" run nonexistent.wdl &
+pid=$!
+sleep 10
+if kill -0 $pid > /dev/null 2>&1; then
+  echo "ERROR: Process still exists"
+  kill $pid
+  exit 1
+fi
```

Comment:
Did something go awry in the rebase/merge?

Reply:
yes, there was a conflict due to testSingleWorkflowRunner.sh being modified both in this branch and develop